src/store.js

```diff
@@ -33,7 +33,8 @@ const state = {
     ps4: [],
     xb1: [],
     switch: [],
-  }
+  },
+  expandInvasions: false
```

Comment:
could you make it part of the invasions state instead of the global state?

just keeps it a bit more organized

Reply:
You mean having something like this?

```js
const state = {
  // some global state parameters...
  invasionsState : {
    expand: false,
    // put some more invasions state parameters here...
  }
}
```

Reply:
like
`components.json`
```json
{
   ....
  "invasions" {
       ....
       "expand": false
   }
}
```

Reply:
once it's moved, i'm 100% happy with this and would love to merge it in